gateway/src/inference/providers/anthropic.rs

```diff
@@ -533,16 +542,26 @@ struct AnthropicResponse {
     usage: AnthropicUsage,
 }
 
-#[derive(Clone, Debug, PartialEq, Serialize)]
-struct AnthropicResponseWithLatency {
+#[derive(Debug, PartialEq, Serialize)]
+struct AnthropicResponseWithMetadata<'a> {
     response: AnthropicResponse,
     latency: Latency,
+    request_body: AnthropicRequestBody<'a>,
```

Comment:
We could call this just `request` since the `response` is also the body.

Reply:
kk